pkg/fscrypt/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM golang:1.12.4-alpine
+
+RUN apk add --no-cache git=2.20.1-r0 gcc=8.3.0-r0 linux-headers=4.18.13-r1 \
+                       libc-dev=0.7.1-r0 make=4.2.1-r2 linux-pam-dev=1.3.0-r0 \
+                       m4=1.4.18-r1
+
+RUN mkdir -p /go/src/github.com/google
+WORKDIR /go/src/github.com/google
+RUN git clone https://github.com/google/fscrypt
```

Comment:
This has to be either a release or a hash of a project. Simply cloning tip of master produces a build that is simply too unreliable.

Reply:
The last release tag is too old (2018), so I've hooked on to a commit hash which I found to be working fine in my testing.